module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -209,6 +209,7 @@ public function buildMultipartBody(
      * none)
      * @param bool                                   $subjectInBody Allow subject to be extracted from body when
      * body text begins with "Subject: " and $body is a string (ignored when $body is an Email object)
+     * @param DataPart[]                             $parts         Data parts to add
```

Comment:
`DataPart` has not been imported -- it's probably worth adding fully-qualified class name or a `use` statement so the full context of this class is clear.